EcoSoapBank/Pickups/NewCartonViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  NewCartonViewModel.swift
+//  EcoSoapBank
+//
+//  Created by Jon Bash on 2020-08-17.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+
+class NewCartonViewModel {
+    @Published var carton: Pickup.CartonContents
+
+    init(carton: Pickup.CartonContents) {
+        self.carton = carton
+    }
+}
+
+extension NewCartonViewModel: Hashable {
+    static func == (lhs: NewCartonViewModel, rhs: NewCartonViewModel) -> Bool {
+        lhs === rhs && lhs.carton == rhs.carton
+    }
+
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(ObjectIdentifier(self))
+    }
+}
```

Comment:
I've had to use this before in code challenges, good to know the ObjectIdentifier thing. You can also use the object identifier for the equality, but I'm sure you know that. Just posting this here for Christopher to note if he hasn't seen it.

``` swift 
ObjectIdentifier(lhs) == ObjectIdentifier(rhs)
```

Reply:
Yeah same re code challenges; I had to do some research at some point to find out it was a thing. I think `===` might even just be syntactic sugar for the expression you wrote? I'm not certain about that, but it would make sense at least.